### ISSUE TITLE
No domain informed

### ISSUE THREAD

**Issue by rzancoper** [IBM/dp-file-uploader#8]
Hi @aharbis 
I see that on the SOMA request there is no domain info, and so, files are pushed to 'default' domain.
In some scenarios (specially in test envs.) users are allowed to upload files only to their own domains, and so, in the way the tool is designed it will certainly fail into an auth. error.
I would suggest adding a command line parameter to get the domain name, and, if omitted, uses the 'default' domain.

**Comment by aharbis** [comment]
Hey @rzancoper thanks for trying this tool out, and thanks for opening the issue.

I definitely can see some value in providing a `--domain "name"` option, so I'm more than happy to take this enhancement request.

In the meantime, you should be able to specify the full path from the default domain's perspective. This is because all application domain sub-directories exist within the default domain's filesystem. So for example if you had a domain `sandbox` and then a sub-directory `foo` in that domain, you could use the tool like this:

```
$ dp-file-uploader my.datapower.com "local:///sandbox/foo" file.txt
```

Alternatively, once this enhancement is implemented, the other form may look like:

```
$ dp-file-uploader --domain "sandbox" my.datapower.com "local:///foo/" file.txt
```

Hope this helps. Let me know if you have any other questions.

**Comment by aharbis** [comment]
@rzancoper This feature is now available in version `0.1.4`.
